Modules/TestQuestionPool/classes/feedback/class.ilAssMatchingQuestionFeedback.php

```diff
@@ -49,8 +49,8 @@ public function getAnswerOptionsByAnswerIndex(): array
     protected function buildAnswerOptionLabel(int $index, $answer): string
     {
         $caption = $ordinal = $index + 1;
-        $caption .= '. <br />"' . $answer->term->text . '" =&gt; ';
-        $caption .= '"' . $answer->definition->text . '"';
+        $caption .= '. <br />"' . $answer->term->getText() . '" =&gt; ';
```

Comment:
This should be like below and the same goes for the next line, right?
```suggestion
        $caption .= '. <br />"' . $answer->getTerm()->getText() . '" =&gt; ';
```